helpers/hjupyter.py

```diff
@@ -73,6 +79,10 @@ def build_run_notebook_cmd(
     :param extra_opts: options for "run_notebook.py", e.g., "--
         publish_notebook"
     """
+    # Importing inside func to avoid error while creating dockerized executable.
+    # TODO(shaunak) -> debug why?
+    import helpers.hgit as hgit
```

Comment:
I think this is because you need to change the PYTHONPATH inside the container.
IIRC there is a similar problem and solution in _llm_transform.py

See if you can easily fix it now, otherwise we'll merge and fix it later.